tests/contrib/elasticsearch/test_elasticsearch.py

```diff
@@ -249,3 +249,21 @@ def test_none_param(self):
             pass
         spans = self.get_spans()
         assert len(spans) == 1
+
+    def test_bulk(self):
```

Comment:
is this meant to be here?

Reply:
whoops! should've stashed my local changes!